writing.md

```diff
@@ -0,0 +1,56 @@
+# Getting Started
+
+This site includes documentation of the tools,
+components, and styles used on OddSite.
+There are also [sample pages](/sample/) available
+for viewing aspects of the site in context --
+including [typography](/sample/type/),
+[colors](/cascading-colors/), [responsive images](/sample/images/), and more.
```

Comment:
1. Is this the right "colors" page to link to? I guess I'm not sure what it's a sample of, or how it is "viewing aspects of the site in context."
2. Do we also want to link to the Events sample page?

Reply:
@jgerigmeyer I wondered how many things we wanted to link to from here, too. I was speaking with @mirisuzanne yesterday about potentially adding a menu like Herman's to those sample pages (and/or macro/filter documented pages)... I wish all of this could live in one place but I know that isn't how it works. So, I added the links I think were helpful overviews of things I thought might be referenced more often than others, but it was purely a guess. I could add links to all the other sample pages, but not having a single navigation for all of this makes it feel disconnected

Reply:
@stacyk That makes sense. Partly I was curious how the `/cascading-colors/` page fits in with "sample pages" here. It reads as an OSS Tool announcement.

Reply:
@jgerigmeyer That page is a bit of a crossover - since it is the product page for CCS, but we show off the tool by rendering the site colors (which used to be on a sample page). So it is both the dynamic colors "sample" AND the CCS tool announcement. If that's strange to link as a sample, it wouldn't be hard to duplicate the demo content.